plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
Could probably do that, or could go back to never storing it and reading it from the image every time. It just seemed like a useful pattern to keep info necessary for starting containers in app dir like the SCALE file is.